lib/restaurant.rb

```diff
@@ -0,0 +1,36 @@
+require_relative 'message_sender'
+class Restaurant
+
+  attr_accessor :menu, :order, :total_amount, :text_is_sent
+
+  def initialize
+    @menu = {
+      :Beef_BBQ => 15,
+      :Chicken_Supreme => 16,
+      :Hawaiian => 14,
+      :Margherita => 12,
+      :Pepperoni => 16,
+      :Texan_BBQ => 15,
+      :Tuna_Sweetcorn => 17,
+      :Vegetable => 16
+      }
+      @order = []
+      @total_amount = 0
+      @text_is_sent = Message_sender.new
+    end
+
+    def dishes_can_be_selected dish, quantity
```

Comment:
Method naming is important - although I appreciate English is not your first language ;o).  The current name of this method suggests its purpose is to validate that a set of dishes can be selected, when what it does is add one dish to an order a number of times.